player/tests/data/quad.vert

```diff
@@ -0,0 +1,18 @@
+#version 450
+
+out gl_PerVertex {
+    vec4 gl_Position;
+};
+
+const vec2 positions[6] = vec2[6](
+    vec2(-1, -1),
+    vec2(-1, 1),
+    vec2(1, 1),
+    vec2(-1, -1),
+    vec2(1, 1),
+    vec2(1, -1)
+);
+
+void main() {
+    gl_Position = vec4(positions[gl_VertexIndex], 0.0, 1.0);
```

Comment:
for a single large triangle, we'd do something like `gl_Position = vec4(gl_VertexIndex == 2 ? 2.0 : -1.0, gl_VertexIndex == 1 ? 2.0 : -1.0, 0.0, 1.0);